IFC4x3/Properties/i/IsBallastTrack_3GWQlZqZbCOhyZObZZzqs$/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IsBallastTrack_3GWQlZqZbCOhyZObZZzqs" Name="IsBallastTrack" UniqueId="d081abe3-d239-4c62-bf23-6258e3f74dbf" PrimaryDataType="IfcBoolean" />
```

Comment:
According to description propose renaming to `HasBallastTrack` or `UsingBallastTrack`.

Reply:
Agree, updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/5eb5d2a2bb92b629c57058b376a6ae07d988a357